pkgs/top-level/all-packages.nix

```diff
@@ -18975,30 +18975,27 @@ with pkgs;
 
   linbox = callPackage ../development/libraries/linbox { };
 
-  ffmpeg_4-headless = callPackage ../development/libraries/ffmpeg/4.nix {
-    inherit (darwin.apple_sdk.frameworks) Cocoa CoreMedia VideoToolbox;
-
-    sdlSupport = false;
-    vdpauSupport = false;
-    pulseaudioSupport = false;
-    libva = libva-minimal;
+  ffmpeg_4-headless = ffmpeg_4.override {
+    ffmpegVariant = "headless";
   };
 
   ffmpeg_4 = callPackage ../development/libraries/ffmpeg/4.nix {
-    inherit (darwin.apple_sdk.frameworks) Cocoa CoreMedia VideoToolbox;
-  };
+    # TODO still necessary?
+    # rtmpdump = null; # Prefer the built-in RTMP implementation
```

Comment:
Following that discussion about splicing and avoiding `inherit ()` in all-packages, it would be more useful to control this with a `...Support` flag.

Reply:
Yes absolutely, I'm just wondering whether we should still favour the built-in rtmp implementation.

Reply:
I decided to just use ours.